docs/whatsnew/1.7.0/1_7_0.rst

```diff
@@ -0,0 +1,47 @@
+Pylake 1.7.0
+============
+
+.. only:: html
+
+Here is an overview of features that were released in Pylake `v1.7.0`.
+
+Improved force calibration
+--------------------------
+
+Pylake 1.7.0 comes with several improvements to the force calibration workflow, making it easier to work with force calibration data.
+It is now possible to recalibrate a force slice using a force calibration item::
+
+  recalibrated_force1x = f.force1x.recalibrate_force(new_calibration)
+
+Force calibration items now have properties containing the raw calibration data in the properties :attr:`~lumicks.pylake.calibration.ForceCalibrationItem.voltage`, :attr:`~lumicks.pylake.calibration.ForceCalibrationItem.driving` and :attr:`~lumicks.pylake.calibration.ForceCalibrationItem.sum_voltage`.
+Recalibrating a force slice with a single calibration setting overridden becomes a simple three lines::
+
+    calibration = file.force1x.calibration[0]
+    recalibrated = calibration.recalibrate_with(hydrodynamically_correct=True)
+    recalibrated_force1x = file.force1x.recalibrate_force(recalibrated)
+
+.. note::
+
+    The properties :attr:`~lumicks.pylake.calibration.ForceCalibrationItem.voltage`, :attr:`~lumicks.pylake.calibration.ForceCalibrationItem.driving` and :attr:`~lumicks.pylake.calibration.ForceCalibrationItem.sum_voltage` are only available if the raw data was exported in Bluelake 2.7.0 or newer with this functionality enabled.
+
+For more information, please refer to the updated :ref:`tutorial<force_calibration_tutorial>`.
+
+Faster bootstraps
+-----------------
+
+Calculating confidence intervals for dwelltime models using bootstrapping can be very time consuming, especially for large datasets.
+To speed up the process, we have added the option to calculate bootstraps in parallel.
+This can be done by setting the `num_processes` parameter in the :meth:`~lumicks.pylake.DwelltimeModel.calculate_bootstrap()` method.
```

Comment:
What sets the maximum of `num_processes`? 
I think users will miss this feature if it is not added to the Tutorial.